src/main/java/pl/javastart/task/Televisor.java

```diff
@@ -0,0 +1,31 @@
+package pl.javastart.task;
+
+public class Televisor {
+    private String onOrOff;
```

Comment:
W przypadku, gdy są możliwe tylko 2 wartości, to najlepiej zastosować typ `boolean`.
Co do nazwy, to też lepiej wybrać taką, które reprezentuje jeden ze stanów. Np: `boolean isOn = true`, oznacza, że jest włączony.